.azure-pipelines/unit_test.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+trigger:
+  - main
+
+pool:
+  vmImage: ubuntu-latest
+strategy:
+  matrix:
+    Python36:
+      python.version: '3.6'
```

Comment:
README states Python 3.7 or higher is a requirement. Should we still check for 3.6?

Reply:
Good idea, will address that.

Reply:
addressed